src/lib/hooks/useStore.js

```diff
@@ -24,10 +31,12 @@ const useStore = create(
 							],
 						};
 					});
-					set({ vinFound: true });
+					set({ vinValidity: true });
+					set({ displayVinError: false });
```

Comment:
you could do this like that:

``set({vinValidity: true, displayVinError: false})``

Reply:
agreed and done :)